public/Invoke-DbaDbAzSqlTip.ps1

```diff
@@ -0,0 +1,209 @@
+function Invoke-DbaDbAzSqlTip {
```

Comment:
I don't really like doing `DbAz` as it seems wrong when this is connecting to Azure SQL **DB**

What about `Invoke-DbaAzSqlDbTip` ?

// @potatoqualitee

Reply:
The naming was weird for me too. I like `Invoke-DbaAzSqlDbTip` - changed unless @potatoqualitee objects 😸

Reply:
Yah tip singular is gr8 ty